release/models/interfaces/openconfig-if-sdn-ext.yang

```diff
@@ -51,7 +51,26 @@ module openconfig-if-sdn-ext {
         `forwarding-viable = false` is not equivalent to
         administratively disabling the interface -- in particular, the
         interface is expected to participate in L2 protocols such as
-        LLDP or LACP even if it blocked from forwarding traffic.";
+        LLDP or LACP even if it blocked from forwarding traffic.
+        Some rules to follow when a port/bundle is set for 
+        Forwarding-viable=False:
+          1. Given that L2 protocols like LACP should be allowed to be 
+          transmitted and received on such ports, a device mustn't consider 
+          such ports in the calculation for minimum number of links required 
+          for an aggregate bundle when such a configuration exists on the box.
+          Hence, the device musn't disable the bundle because the minimum
+          number of member links requirement is not met for the bundle.
+          2. While it is required that L2 protocols like LLDP, LACP must
+          be allowed for transmit and receive on such ports/bundles, CLNS
+          protocol is an exception and must be treated as per the requirement
+          for L3 protocols below.
```

Comment:
I'd argue that CLNS _is_ a layer-3 protocol.

But to keep things simple and straightforward, maybe we can just state that IS-IS is considered a L3 protocol for the purposes of this definition?

Reply:
Done